releng/com.espressif.idf.target/com.espressif.idf.target.target

```diff
@@ -114,5 +109,23 @@
 		   <repository location="http://download.eclipse.org/nebula/releases/latest"/>
 		   <unit id="org.eclipse.nebula.widgets.xviewer.feature.feature.group" version="0.0.0"/>
 	   </location>
+	   <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+		   <repository location="https://download.eclipse.org/embed-cdt/releases/6.1.2/p2/"/>
+		   <unit id="org.eclipse.embedcdt.codered.feature.group" version="6.1.2.202102181132"/>
+		   <unit id="org.eclipse.embedcdt.codered.source.feature.group" version="6.1.2.202102181132"/>
+		   <unit id="org.eclipse.embedcdt.debug.gdbjtag.feature.group" version="6.1.2.202102181132"/>
+		   <unit id="org.eclipse.embedcdt.debug.gdbjtag.jlink.feature.group" version="6.1.2.202102181132"/>
+		   <unit id="org.eclipse.embedcdt.debug.gdbjtag.jlink.source.feature.group" version="6.1.2.202102181132"/>
+		   <unit id="org.eclipse.embedcdt.debug.gdbjtag.openocd.feature.group" version="6.1.2.202102181132"/>
+		   <unit id="org.eclipse.embedcdt.debug.gdbjtag.openocd.source.feature.group" version="6.1.2.202102181132"/>
+		   <unit id="org.eclipse.embedcdt.debug.gdbjtag.pyocd.feature.group" version="6.1.2.202102181132"/>
+		   <unit id="org.eclipse.embedcdt.debug.gdbjtag.pyocd.source.feature.group" version="6.1.2.202102181132"/>
+		   <unit id="org.eclipse.embedcdt.debug.gdbjtag.qemu.feature.group" version="6.1.2.202102181132"/>
+		   <unit id="org.eclipse.embedcdt.debug.gdbjtag.qemu.source.feature.group" version="6.1.2.202102181132"/>
+		   <unit id="org.eclipse.embedcdt.debug.gdbjtag.source.feature.group" version="6.1.2.202102181132"/>
+		   <unit id="org.eclipse.embedcdt.doc.user.feature.group" version="6.1.2.202102181132"/>
+		   <unit id="org.eclipse.embedcdt.doc.user.source.feature.group" version="6.1.2.202102181132"/>
+		   <unit id="org.eclipse.embedcdt.feature.group" version="6.1.2.202102181132"/>
+	   </location>
```

Comment:
Could we add only required dependencies otherwise build time will increase for fetching necessary plugins?

For example, org.eclipse.embedcdt.debug.gdbjtag.pyocd.feature.group is not required